pytorch_lightning/trainer/connectors/profiler_connector.py

```diff
@@ -50,7 +50,7 @@ def on_trainer_init(self, profiler: Union[BaseProfiler, str]):
             else:
                 raise ValueError(
                     "When passing string value for the `profiler` parameter of"
-                    " `Trainer`, it can only be 'simple' or 'advanced'"
+                    " `Trainer`, it can only be 'simple', 'advanced' or 'pytorch'"
                 )
```

Comment:
use `PROFILERS.keys()` from above so this error message is always in sync with what's supported